backend/db.js

```diff
@@ -39,6 +41,11 @@ const Todos = {
 
   toggleComplete: (id) => {
     return db.none(TOGGLE_COMPLETE, [id])
+  },
+  // lowTodoParams = [lowTodoId, higherTodoPirority]
+  // highTodoParams = [highTodoId, lowTodoPirority]
+  swapPriorities: ( lowTodoParams, highTodoParams ) => {
```

Comment:
Thanks Punit! 
I guess I did hide the information cause it just becomes redundant since the swapping 'math' is being taken care of on the front end. So I guess the comment is redundant, and my thinking there was leaving it in so I didnt forget when I explained it or if someone else looked at my artifact and wanted to follow along.